src/Makefile.OCaml

```diff
@@ -476,6 +460,7 @@ clean::
 	-$(RM) system/generic/*.cm[iox] system/generic/*.{o,obj} system/generic/*~
 	-$(RM) system/win/*.cm[iox] system/win/*.{o,obj} system/win/*~
 	-$(RM) fsmonitor/*.cm[iox] fsmonitor/*.{o,obj}
+	-$(RM) .depend.dot.tmp
```

Comment:
No need, as there is `*.tmp` already. But `DEPENDENCIES.ps` should be added here.

Reply:
*.tmp does not match .foo.tmp

Reply:
> *.tmp does not match .foo.tmp

You are right, of course. It should be the other way around, the tmp file name does not need to begin with a dot.